code/datums/mutations/body.dm

```diff
@@ -224,17 +224,35 @@
 
 /datum/mutation/human/strong
 	name = "Strength"
-	desc = "The user's muscles slightly expand."
+	desc = "The user's muscles slightly expand, allowing them to move heavy objects easily."
 	quality = POSITIVE
 	text_gain_indication = "<span class='notice'>You feel strong.</span>"
 	difficulty = 16
+	instability = 25
+
+/datum/mutation/human/strong/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.slowed_by_drag = FALSE // Don't get slowed by dragging lockers or mobs
+
+/datum/mutation/human/strong/on_losing(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.slowed_by_drag = initial(owner.slowed_by_drag)
```

Comment:
slowed_by_drag should be made a trait for futureproofing

Reply:
i'll look into it